### DISCUSSION TITLE
Stanford STATS214 / CS229M: Machine Learning Theory - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#489] [Announcements]
# Stanford STATS214 / CS229M: Machine Learning Theory - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%E8%BF%9B%E9%98%B6/CS229M/

**Comment by FuRuF-11** [comment]
请问这门课程的视频有什么观看渠道吗

**Reply by CheyneeeWww** [reply]
youtube搜CS229M

**Comment by Skaiyin** [comment]
请问哪里可以找到作业呢

**Comment by shaoyaoqw** [comment]
圣殿。。加油！

**Comment by ChunYunMuXue** [comment]
这门课比较痛苦的点是油管的视频第一章没有板书

**Comment by Zane-Liao** [comment]
这门课的板书地址: https://scribbletogether.com/whiteboard/0AB9BF4D-43DA-4C3B-A65C-C1B2D9C32C5E
